go/oasis-node/cmd/registry/entity/entity.go

```diff
@@ -358,8 +359,17 @@ func loadOrGenerateEntity(dataDir string, generate bool) (*entity.Entity, signat
 		return nil, nil, fmt.Errorf("loadOrGenerateEntity: sanity check failed: one or more unsafe debug flags set")
 	}
 
-	// TODO/hsm: Configure factory dynamically.
-	entitySignerFactory := fileSigner.NewFactory(dataDir, signature.SignerEntity)
+	var entitySignerFactory signature.SignerFactory
```

Comment:
This seems like duplicated code from `LoadEntity` in `cmd/common`. Instead of duplicating the code move it into a separate function in `cmd/common/common` and use that function in both places.